docs/spl-docs/splunk-commands.md

```diff
@@ -17,3 +17,5 @@ _The following list includes the supported SPL Commands._
 - [top command](top-command.md)
 - [transaction command](transaction-command.md)
 - [where command](where-command.md)
+- [streamstats command](streamstats-command.md)
+- [gentimes command](gentimes-command.md)
```

Comment:
nit: these should be placed alphabetically

Reply:
addressed in the latest commit.